pages/index.tsx

```diff
@@ -7,48 +7,57 @@ import { use, useEffect, useMemo } from 'react'
 import { GoogleMap, useLoadScript, Marker, useJsApiLoader } from '@react-google-maps/api'
 import { env } from 'process'
 import { useState } from 'react';
-import FindLocation from '@/hooks/FindLocation' 
-
+import FindLocation from '@/hooks/FindLocation'
+import Map from '@/components/Map'
 
 export default function Home() {
+
   //These are the two states used to get our location for centering
-  const [location, setLocation] = useState()
+  const [location, setLocation] = useState(null)
   const [hasLoaded, setHasLoaded] = useState(false)
+  const [showInfo, setShowInfo] = useState(false)
 
-// this sets our location State using this function
+  // this sets our location State using this function
   function setUserLocation() {
     return FindLocation().then((value) => {
       setLocation(value)
       return value
     })
   }
 
-//This is a useEffect used to make sure that the users location is grabbed only once when the page is rendered
+  //This is a useEffect used to make sure that the users location is grabbed only once when the page is rendered
   useEffect(() => {
+    fetch('api/pointsOfInterests').then((data) => {
+      console.log(data)
+    })
     if (!hasLoaded) {
-      setUserLocation()
+      // setUserLocation()
```

Comment:
[pebble] You can add this back. We commented it out for testing

Reply:
Fixed